server/channels/store/sqlstore/user_store.go

```diff
@@ -2444,7 +2445,7 @@ func (us SqlUserStore) GetUserReport(filter *model.UserReportOptions) ([]*model.
 		}
 
 		parentQuery = us.getQueryBuilder().
-			Select("*").
+			Select("data.*").
```

Comment:
I'll likely allow an exclusion for this when we enable the linter, since it's not the same as the usual `SELECT *`.